client/asset/eth/nodeclient_harness_test.go

```diff
@@ -143,6 +139,33 @@ func newRedeem(secret, secretHash [32]byte) *asset.Redemption {
 	}
 }
 
+// waitForReceipt waits for a tx. This is useful on testnet when a tx may be "missing"
+// due to reorg. Wait for a few blocks to find the main chain and hopefully our tx.
+func waitForReceipt(t *testing.T, nc *nodeClient, hash common.Hash) (*types.Receipt, error) {
+	t.Helper()
+	// Waiting as much as five blocks.
+	timesUp := time.After(time.Duration(5*secPerBlock) * time.Second)
```

Comment:
looks like there is no need for type-casting here: I think `time.After(5*secPerBlock * time.Second)` should work just fine,

also, maybe converting `secPerBlock` type to `time.Duration` where variable is defined (making `secPerBlock = time.Second`) would be result into its usage being cleaner, otherwise multiple places require this `* time.Second` part:

<img width="1344" alt="image" src="https://user-images.githubusercontent.com/7031569/177004421-3535aad6-399b-4760-b7a7-4bcaa746380c.png">

Reply:
It seems I do need to cast: `./nodeclient_harness_test.go:147:24: invalid operation: 5 * secPerBlock * time.Second (mismatched types int and time.Duration)`

Maying `secPerBlock` a `time.Duration` is good tho.